### PR TITLE
Commenting edges.py

### PR DESCRIPTION
**Pull request by Norric1Admin**
Adding comments to the .py script to explain how disaggregations are read from .csv files
These were written by @ONSBen

### CODE REVIEW

**Comment by brockfanning** [comment]
Very nice - I will give @dougmet the first crack at a review since he is the most familiar.

File: sdg/edges.py [hunk 1]
```diff
@@ -46,6 +44,9 @@ def x_without_y(x, y):
         x (pandas Series): Left hand column
         y (pandas Series): Right hand column
     """
+    # returns true if there are any cases where an element in
```
Comment (dougmet) [inline sdg/edges.py]: These comments should go in the doc string
```suggestion
    """
     Args:
        x (pandas Series): Left hand column
        y (pandas Series): Right hand column
    
    Returns:
        bool: True if there are any cases where an element in the y column is 
                 empty (is a nan) and the corresponding element in the x column
                 is not empty
    """
```

File: sdg/edges.py [hunk 2]
```diff
@@ -116,13 +146,10 @@ def prune_grand_parents(edges):
 
 def edge_detection(inid, df):
     """Check dependencies between columns and write out the edges
-
```
Comment (dougmet) [inline sdg/edges.py]: Any particular reason to remove all the blank lines?

Comment (ONSBen) [reply sdg/edges.py]: I don't know why that happened - I usually like plenty of space around my comments, so I don't think it was intentional. I might have changed the file format a few times while finding the best way to edit it, which could have removed the blank lines.

File: sdg/edges.py [hunk 3]
```diff
@@ -85,28 +99,44 @@ def detect_all_edges(inid, df):
...
 def prune_grand_parents(edges):
     """Prune edges that shortcut a parent-child relationship
-
     Args:
         edges (DataFrame): The edges data frame
-
     Returns:
         The data frame with grand parent edges removed
     """
+    # if A is a parent of B and B is a parent of C, then the above code will
```
Comment (dougmet) [inline sdg/edges.py]: this section in the doc string too

**Comment by dougmet** [comment]
@ONSBen @Norric1Admin would you mind if I put this in a new branch and submit the PR? I can't make edits to the forked branch but thought I could quickly make some of these changes for you to review.

**Comment by dougmet** [comment]
Closing for #10

**Comment by Norric1Admin** [comment]
Yes no problem - @ONSBen is back next Monday so he may have a comment too. This all relates to @dougmet's recent work on https://github.com/open-sdg/open-sdg/issues/86, which is much appreciated.

**Comment by dougmet** [comment]
No worries. I just thought it's quicker for me to do the whitespace stuff. I won't change the wording on the comments. Just want my docstrings all neat and tidy!

**Pull request closed without merging**